src/components/icon/icon.tsx

```
@@ -649,16 +645,6 @@ export const EuiIcon: SFC<Props> = ({
   );
 };
 
-function checkValidColor(color: string, type: string) {
```

With this gone it will just take any string now right? Basic a silent fail if it's malformed? Is everyone OK with that?

---

Yep! It will just fail on the CSS side as it would if you just normally passed it via `style="color: [color'"`

---

Until the recent TSXification of the component this was a proptype warning, which apparently was ignored by TSVB previously anyway :)